packages/edit-site/src/index.js

```diff
@@ -49,6 +49,8 @@ export function initialize( id, settings ) {
 		fetchLinkSuggestions( search, searchOptions, settings );
 	settings.__experimentalFetchRichUrlData = fetchUrlData;
 	settings.__experimentalSpotlightEntityBlocks = [ 'core/template-part' ];
+	// Feature flag for the new menu sidebar which isn't stable yet.
+	settings.__experimentalNewMenuSidebar = false;
```

Comment:
Im curious why this is hardcoded here as `false` as opposed to initialized as `false` on the php side for the settings that are passed into this init function.  With the former, the nav sidebar is always there without making direct changes to the build.  With the latter, we could filter this setting and have it either way?

Reply:
This is just a temporary solution and I don't want to make it too easy to change that. It wouldn't be here for too long anyway. Also, I think you could do `wp.data.dispatch( editSiteStore ).updateSettings()` to switch this flag using JS.